### PR TITLE
Show cpu and memory usage in a table

### LINKED ISSUE #1343: Nodes list on Cluster Overview Changes to List mode
As [Sebastian](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#sebastian---tech-lead) and [Mathew](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#matthew---cloud-native-software-engineer) I would like to be able to view clusters that have 10+ nodes.

**Acceptance Criteria**

- [ ] We will introduce a list view mode for nodes with inline graphs for CPU, Memory and Disk Space
- [x] Inline Graphs have tooltips showing the values
- [ ] If prometheus is present, we will use prometheus data as input if not Kubernetes API data as input
- [x] We show status| age | k8s version as list items
- [x] Deep links goes to Node Detail
- [x] We will use the theme colours and we will support dark mode
- [x] The inline graphs can be used by other extensions ( reusable )
- [ ] Follow Project DOD

**UI Guides**

- 
image

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

Resolves #1343

### CODE REVIEW

File: core-ui/src/components/Clusters/views/ClusterOverview/ClusterNodes/ClusterNodes.js [hunk 1]
```diff
@@ -1,75 +1,155 @@
...
+      <NodeHeader nodeName={entry.metadata?.name} />,
+      cpu ? (
+        <ProgressBar
+          percentage={cpu.percentage}
+          tooltip={{
+            content: t('cluster-overview.tooltips.cpu-used', {
+              percentage: cpu.percentage,
+            }),
+            position: 'bottom',
+            color: 'var(--sapIndicationColor_5)',
```
Comment (akucharska) [inline core-ui/src/components/Clusters/views/ClusterOverview/ClusterNodes/ClusterNodes.js:99]: Can we use the same color that we use for in stats-graph CPU: `sapIndicationColor_7`?

**Review by akucharska — COMMENTED** [review]
I have one suggestion: can we keep the same colors as on graphs?

**Review by grego952 — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**